Makefile

```diff
@@ -25,7 +25,7 @@ else
 endif
 
 EXTENSION = pg_shard
-DATA = pg_shard--1.0.sql
+DATA = pg_shard--1.1.sql pg_shard--1.0--1.1.sql
```

Comment:
One last question then. The create-extension command also supports a version. If I specify create-extension version 1.0, would that work? (And create 1.0 without the UDF?)

After checking on that, and assuming we've tested 1.0-1.1 upgrade and a 1.1 fresh install, you're good to :shipit: